README.md

```diff
@@ -30,11 +30,11 @@ For a list of contributors, [click here](https://github.com/OpenEnergyDashboard/
 
 This project is licensed under the MPL version 2.0.
 
-See the full licensing agreement [here](LICENSE.txt)
+See the full licensing agreement [here](License.txt)
```

Comment:
I'm not sure if this is intentional, but I followed the link, and it displays a 404 warning.

Reply:
Interesting and good catch. I changed the case to be correct but it did not seem to take so I just manually updated on the repo. I updated the PR so it should now be correct.